_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -81,38 +100,65 @@ plot_rt_fn <- function(region_name){
               alpha = 0.9, size = 1) +
     ggplot2::geom_hline(yintercept = 1, linetype = 2) +
     ggplot2::coord_cartesian(ylim = c(NA, 1.5 * max(region_summary[["median"]]))) +
```

Comment:
also it looks like the uncertainty issue has cleared up so these can be made free again to reduce whitespace in the plot.